third_party/catch/catch.hpp

```diff
@@ -13416,13 +13416,7 @@ namespace Catch {
 
         void renderTestProgress(int current_test, int total_tests, std::string next_test) {
             double progress = (double) current_test / (double) total_tests;
-            int render_width = 80;
```

Comment:
in CLI we probably want to use the width of the terminal, but remove the truncation when not ran in a CLI setting

Reply:
Another helpful thing might be to show the *end* of the test instead of the *beginning* when truncating since that is generally the most useful part (contains the test name).